src/addEditErase.js

```diff
@@ -0,0 +1,62 @@
+/*eslint-disable*/
+import { save, load } from './data.js';
+/* eslint-enable */
+const theBigList = document.querySelector('.todos');
+
+export function antiShowAll(element) {
+  while (element.lastElementChild) {
+    element.removeChild(element.lastElementChild);
+  }
+}
+
+export function elimanateCompleteds() {
+  const actualtodo = load();
+  const result = actualtodo.filter((activity) => (activity.completed === false));
+  for (let i = 0; i < result.length; i += 1) {
+    result[i].index = i;
+  }
+  save(result);
+}
+
+export function addActivity(text) {
+  const actualtodo = load();
+  const { length } = actualtodo;
+  const newActivity = {
+    description: text,
+    completed: false,
+    id: Date(),
```

Comment:
Please change the way you are indexing the activities.

<img width="897" alt="Screen Shot 2022-07-13 at 9 10 11" src="https://user-images.githubusercontent.com/82426682/178754294-af6fd887-a8f5-4525-a6fe-a161244bf172.png">